crates/biome_cli/src/execute/migrate/eslint_any_rule_to_biome.rs

```diff
@@ -186,18 +186,6 @@ pub(crate) fn migrate_eslint_any_rule(
                 .get_or_insert(Default::default());
             rule.set_level(rule_severity.into());
         }
-        "@typescript-eslint/no-throw-literal" => {
```

Comment:
https://typescript-eslint.io/rules/only-throw-error/

It's deprecated
![image](https://github.com/user-attachments/assets/5e36edb6-1a2c-481e-a102-f9f7003eb30e)

Reply:
@Conaclos @Jayllyz 
I don't know why.
But, maybe codegen deleted that when I solve confrict.​​​​​​​​​​​​​​​​

Someone deleted this deprecated code, but codegen wasn't performed at that time. 
Then, when codegen was executed with this change that incorporated that deletion, it ended up being included in this change... or something like that?

Reply:
@Jayllyz 
Thank you for the supplemental information!

Reply:
It looks like a bug in the code generation. We will have to investigate the case. For now, it is ok because it is not related to the PR.

Reply:
@Conaclos 
> We will have to investigate the case. For now, it is ok because it is not related to your PR.

Okay, thank you.
Now, may I assume that this has been resolved?

Reply:
> Now, may I assume that this has been resolved?

Fixed by https://github.com/biomejs/biome/pull/3692

Reply:
@Jayllyz 
Thanks a lot!
I'll change it to resolved.